pom.xml

```diff
@@ -32,12 +32,12 @@ limitations under the License.
         </license>
     </licenses>
     <properties>
-        <scala.version>2.11.8</scala.version>
+        <scala.version>2.11.12</scala.version>
```

Comment:
is this backward compatible with earlier versions of scala and spark?

any reason we are bumping the spark/scala version?

Reply:
Yes - it is binary compatible (scala increases the minor version - e.g. 2.11 --> 2.12) for breaking changes. I made this specific version upgrade because it contains useful security updates: https://scala-lang.org/news/security-update-nov17.html